fuzzing/libs/lib_cxng.cmake

```diff
@@ -0,0 +1,55 @@
+include_guard()
+include(${BOLOS_SDK}/fuzzing/macros/macros.cmake)
+include(${BOLOS_SDK}/fuzzing/mock/mock.cmake)
+
+file(
+  GLOB
```

Comment:
We could remove the use of file/GLOB if we need to specify all the files anyway.
If you did this because there is some files that you did not want to include, you can do your globing and then use list(REMOVE_ITEM ) to remove what you needed to exclude